monolith/datasources/reviews.js

```diff
@@ -18,9 +18,9 @@ class ReviewsDb extends DataSource {
     const config = {
       username: 'root',
       password: null,
-      database: 'database_development',
+      database: 'dsatabase_development',
```

Comment:
Are these names intentional? I wonder if it will strike us as a typo in the future?

Reply:
I think this is a typo from me, oops. Good catch!!